spec/views/devise/mailer/confirmation_instructions.html.slim_spec.rb

```diff
@@ -29,10 +26,8 @@
   end
 
   it 'includes a request to not reply to this messsage' do
-    assign(:resource, build_stubbed(:user, confirmed_at: Time.zone.now))
-    render
-
-    expect(rendered).to have_content 'PLEASE DO NOT REPLY TO THIS MESSAGE'
+    expect(last_email.parts.first.body.raw_source).
+      to have_content 'Please do not reply to this message.'
```

Comment:
@monfresh This test is failing because the text "please do not reply to this message" has been moved into the user_mailer layout.  This test seems to only look at the rendered content of the actual confirmation_instructions.html.slim file which no longer includes the text.  I used last_email in order to see the entire email, but that may not be the correct way to do it.